### ISSUE TITLE
Back-end Systems

### ISSUE THREAD

**Issue by sarahruggless** [18F/bpa-opm-eqip#33]
# Question/Comment on e-QIP Prototype RFQ

## Name and affiliation
Sarah Ruggless, Flexion, VP - Public Sector 

## Section of RFQ documents
PWS section 2.0 OBJECTIVES
Section 2.0 states: OPM and NBIB are seeking the creation of a new front-end/user interface prototype of the e-QIP system.  

## Question/Comment
In addition to back-end systems that will provide validation functions, is the completed data structure submitted to a back-end SSOT for persistence and full validation? Does this back-end persistence and validation system and API already exist? Should we expect to make or request changes to this API to support the new front-end?

**Comment by billygriffin** [comment]
No. There is no pre-existing portion of this deliverable. Form validations (whether done client-side or server-side) will be determined based on user research.
